examples/unstructured/frontend.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# ignore "local variable '...' is assigned to but never used" for the entire file
+# flake8: noqa: F841
+from gt_frontend.frontend import GTScriptCompilationTask
+from gt_frontend.gtscript import (
+    FORWARD,
+    Edge,
+    Field,
+    Mesh,
+    Vertex,
+    computation,
+    interval,
+    location,
+    vertices,
+)
+
+from gtc import common
+
+
+dtype = common.DataType.FLOAT64
+
+
+def edge_reduction(mesh: Mesh, edge_field: Field[Edge, dtype], vertex_field: Field[Vertex, dtype]):
```

Comment:
The first 3 examples are not used. I propose to remove them (and make them tests if needed).

Reply:
After they are removed, I'd move the file to the fvm examples folder.

Reply:
I turned these into tests (for the code generation) and removed the file.